scripts/tests/Slice/generation.py

```diff
@@ -8,7 +8,7 @@ class SliceGenerationTestCase(ClientTestCase):
     def runClientSide(self, current):
         current.write("testing list-generated... ")
 
-        slice2java = SliceTranslator("slice2java")
+        slice2java = SliceTranslator("slice2java", quiet=True)
```

Comment:
Unrelated to the PR, but passing `--list-generated` causes the `depends` content to be written to `stdout`.
Passing `true` here prevents it from getting dumped into the testing output.